app/views/games/categories/leaderboards/sum_of_bests/index.slim

```diff
@@ -3,36 +3,38 @@
   h2 Fastest sum of best
   h4 = link_to @category.game, game_path(@game), class: 'stealth-link game-link'
 article data-turbolinks-temporary=true
-  = render partial: 'shared/category_tabs', locals: {game: @game, current_category: @category, link_type: :sum_of_best}
-  - runners = @category.runners.joins(:runs).includes(:runs).where('runs.realtime_sum_of_best_ms > 0').order('runs.realtime_sum_of_best_ms asc')
-  - if runners.empty?
-    center
-      i Not enough data in this category to display a sum-of-best leaderboard.
-  - else
-    .card
-      table.card-body.table.table-striped.mb-0
-        thead
-          tr
-            th
-            th.align-left Runner
-            th.align-left Sum of best
-            th.align-left
-              small PB
-            th.align-left
-              small
-                | Potential save
-                a.tip< title='(PB - sum of best)'
-        tbody
-          - i = 0
-          - runners.each do |runner|
-            - pb = runner.pb_for(@category)
-            tr id=[:run, pb.id]
-              td.align-right = i += 1
-              td.align-left = link_to runner, runner, class: 'user-link'
-              td.align-left = format_milliseconds(pb.sum_of_best_ms(Run::REAL))
-              td.align-left
-                small = link_to format_milliseconds(pb.duration_ms(Run::REAL)), pb
-              td.align-left
-                small = format_milliseconds(pb.duration_ms(Run::REAL) - pb.sum_of_best_ms(Run::REAL))
-    center
-      small = link_to 'normal run list', game_category_path(@game, @category), class: 'category-link'
+  .card
+    .card-header
+      = render partial: 'shared/category_tabs', locals: {game: @game, current_category: @category, link_type: :sum_of_best}
+    - runners = @category.runners.joins(:runs).includes(:runs).where('runs.realtime_sum_of_best_ms > 0').order('runs.realtime_sum_of_best_ms asc')
+    - if runners.empty?
+      center
+        i Not enough data in this category to display a sum-of-best leaderboard.
+    - else
+      .table-responsive
+        table.card-body.table.table-striped.mb-0
+          thead
+            tr
+              th
+              th.align-left Runner
+              th.align-left Sum of best
+              th.align-left
+                small PB
+              th.align-left
+                small
+                  | Potential save
+                  a.tip< title='(PB - sum of best)'
```

Comment:
This should probably be a FA icon as well if you want to keep the theme of using them like on the stats page.